schema/deploy/operations/test-quota.sql

```diff
@@ -0,0 +1,40 @@
+-- Deploy seattleflu/id3c-customizations:operations/test-quota to pg
+
+begin;
+
+set local search_path to operations;
+
+create table operations.test_quota (
+    name text not null,
+    timespan tstzrange not null,
+
+    /* Using a reserved word is slightly annoying because we'll have to quote
+     * it everywhere… but I dunno, the other words I can think of aren't quite
+     * as descriptive.  Going with it for now!
+     *   -trs, 17 Sept 2020
+     */
+    "limit" integer check ("limit" >= 0) not null,
```

Comment:
Brainstorming - how about replacing limit with maximum, upper bound, or quota? I'm not a fan of reserved words as column names either

Reply:
Fair! I'd love better options. "Limit" makes most sense to me and is used in other quota systems I've used, but reserved words are a little awkward (but also we're unlikely to be manually using them much).

I didn't use `quota` because "quota" conceptually encompasses the whole row/set of rows for me, not just the limit/max/upper bound, and this is why "quota" is in the table name.

I like `maximum` or `max`. That makes sense as a more specific "limit", since limit can be either a minimum or a maximum.

Reply:
Renamed to `max` in the latest repush.